diff --git a/node_modules/decentraland-ecs/dist/index.d.ts b/node_modules/decentraland-ecs/dist/index.d.ts
index a9406b8..0aebbf3 100644
--- a/node_modules/decentraland-ecs/dist/index.d.ts
+++ b/node_modules/decentraland-ecs/dist/index.d.ts
@@ -4543,7 +4543,7 @@ export declare enum Space {
 export declare class SphereShape extends Shape {
 }
 
-declare class Subscription {
+export declare class Subscription {
     fn: (e: LocalActionButtonEvent) => void;
     useRaycast: boolean;
     constructor(fn: (e: LocalActionButtonEvent) => void, useRaycast: boolean);
